overlord/fdestate/dbx_test.go

```diff
@@ -470,6 +486,9 @@ func (s *fdeMgrSuite) TestEFIDBXUpdateAndCleanupRunningAction(c *C) {
 }
 
 func (s *fdeMgrSuite) TestEFIDBXUpdateAndUnexpectedStartupAction(c *C) {
+	mountinfo := `26 27 8:3 / %s/var/lib/snapd/save rw,relatime shared:7 - ext4 /dev/fakedevice0p1 rw,data=ordered`
+	defer osutil.MockMountInfo(fmt.Sprintf(mountinfo, dirs.GlobalRootDir))()
```

Comment:
no need to copy this over and over again, this can be moved to a helper on `fdeMgrSuite`

Reply:
Done.